Cython/Compiler/Lexicon.py

```diff
@@ -77,7 +77,7 @@ def prefixed_digits(prefix, digits):
     punct = Any(":,;+-*/|&<>=.%`~^?!@")
     diphthong = Str("==", "<>", "!=", "<=", ">=", "<<", ">>", "**", "//",
                     "+=", "-=", "*=", "/=", "%=", "|=", "^=", "&=",
-                    "<<=", ">>=", "**=", "//=", "->", "@=")
+                    "<<=", ">>=", "**=", "//=", "->", "@=", "&&")
```

Comment:
Is this OK? I was afraid that this would be too intrusive, but it might become necessary if Cython wants to support rvalue references more broadly going forward

Reply:
I created a PR for the error reporting in #3858.